tests/playground/ExchangePriceRepo/getPrice.js

```diff
@@ -2,17 +2,21 @@ const PriceRepoImpl = require('../../../src/repositories/PriceRepo/PriceRepoImpl
 
 const EXCHANGE_PRICE_FEED_STRATEGIES_DEFAULT = {
   strategy: 'sequence', // TODO: More strategies can be implemented. i.e. averages, median, ponderated volumes, ...
-  feeds: ['binance', 'huobi', 'kraken', 'bitfinex']
+  feeds: ['binance', 'huobi', 'kraken', 'bitfinex', 'liquid']
 }
 
 const EXCHANGE_PRICE_FEED_STRATEGIES = {
   'WETH-OMG': {
     strategy: 'sequence',
-    feeds: ['binance', 'huobi', 'bitfinex']
+    feeds: ['binance', 'huobi', 'bitfinex', 'liquid']
   },
   'WETH-RDN': {
     strategy: 'sequence',
-    feeds: ['huobi', 'binance', 'bitfinex']
+    feeds: ['huobi', 'binance', 'bitfinex', 'liquid']
+  },
+  'WETH-GEN': {
+    strategy: 'sequence',
+    feeds: ['huobi', 'binance', 'bitfinex', 'liquid']
```

Comment:
Are you sure the feeds you need for WETH-GEN and they are in the right order?

If for example liquid is the one more trustworthy for your token, it should be the first one. If any of the exchanges doesn't have liquidity for GEN, you should omit it from the list.